### DISCUSSION TITLE
Half star for ratings

### DISCUSSION THREAD

**Discussion by Suya1671** [navidrome/navidrome#3074] [Ideas]
It would be nice to be able to have half stars for rating. Not sure if OpenSubsonic's API allows fractional ratings but it would be quite nice to have since I have a lot of songs that I'd rate 3.5 or 4.5 because their rounding in either direction don't feel right

**Comment by Fraccs** [comment]
I feel the same way. I also wouldn't mind if half-star ratings were an option. Perhaps @deluan could clarify whether this is an issue related to OpenSubsonic or if it's specific to the Navidrome implementation.

**Reply by kgarner7** [reply]
It's defined as an int (1-5, inclusive, and 0 to clear) in the original Subsonic specification. See `UserRating` in https://www.subsonic.org/pages/inc/api/schema/subsonic-rest-api-1.16.1.xsd

**Comment by deluan** [comment]
We have an old discussion about enhancing the user rating support. I'm trying to revive the discussion, and also added the support for half-star: https://github.com/opensubsonic/open-subsonic-api/discussions/17#discussioncomment-11879216

Feel free to show support for the idea in that forum, as users requesting features usually have (or should have) more impact than devs opinions.
